ospf6d/ospf6_lsdb.h

```diff
@@ -92,7 +93,8 @@ enum ospf_lsdb_show_level {
 
 extern void ospf6_lsdb_show(struct vty *vty, enum ospf_lsdb_show_level level,
```

Comment:
This function definition can be removed from here to ospf6d.c as it is not used in ospf6_lsdb.c 
So the include of  "lib/json.h" in ospf6_lsdb.h file can be removed.

Reply:
ospf6_lsdb_show function is present in ospf6_lsdb.c, so it's declaration should be in ospf6_lsdb.h right? Or are you asking to move the whole function itself from ospf6_lsdb.c to ospf6d.c? That could also be done. Just want to confirm. Thanks

Reply:
Correct , I think it can be moved to ospf6d.c as it is used in that file only.

Reply:
Done